docs/index.rst

```diff
@@ -8,45 +8,43 @@ PlasmaPy Documentation
 
 .. raw:: html
 
-    <div style="width: 30%;" class="sphx-glr-thumbcontainer" tooltip="Let&#x27;s try to look at ITER plasma conditions using the physics subpackage. ">
-
-.. only:: html
-
-    .. figure:: /auto_examples/images/thumb/sphx_glr_plot_physics_thumb.png
-
-        :ref:`sphx_glr_auto_examples_plot_physics.py`
-
-.. raw:: html
-
+  <div class="sphx-glr-thumbcontainer">
+    <div class="figure align-center">
+      <img alt="thumbnail" src="./_images/notebooks_physics_16_0.png" />
+      <p class="caption">
+        <span class="caption-text">
+          <a class="reference internal" href="notebooks/physics.html">
+            <span class="std std-ref">Analysing ITER parameters</span>
+          </a>
+        </span>
+      </p>
     </div>
-
-.. raw:: html
-
-    <div style="width: 30%;" class="sphx-glr-thumbcontainer" tooltip="Let&#x27;s import some basics (and PlasmaPy!) ">
-
-.. only:: html
-
-    .. figure:: /auto_examples/images/thumb/sphx_glr_plot_dispersion_function_thumb.png
-
-        :ref:`sphx_glr_auto_examples_plot_dispersion_function.py`
-
-.. raw:: html
-
+  </div>
+  <div class="sphx-glr-thumbcontainer">
+    <div class="figure align-center">
+      <img alt="thumbnail" src="./_images/notebooks_dispersion_function_9_0.png" />
+      <p class="caption">
+        <span class="caption-text">
+          <a class="reference internal" href="notebooks/dispersion_function.html">
+            <span class="std std-ref">The plasma dispersion function</span>
+          </a>
+        </span>
+      </p>
     </div>
-
-.. raw:: html
-
-    <div style="width: 30%;" class="sphx-glr-thumbcontainer" tooltip="Let&#x27;s analyze a few Langmuir probe characteristics using the diagnostics.langmuir subpackage. F...">
-
-.. only:: html
-
-    .. figure:: /auto_examples/images/thumb/sphx_glr_plot_langmuir_analysis_thumb.png
-
-        :ref:`sphx_glr_auto_examples_plot_langmuir_analysis.py`
-
-.. raw:: html
-
+  </div>
+  <div class="sphx-glr-thumbcontainer">
+    <div class="figure align-center">
+      <img alt="thumbnail" src="./_images/notebooks_langmuir_analysis_6_1.png" />
+      <p class="caption">
+        <span class="caption-text">
+          <a class="reference internal" href="notebooks/langmuir_analysis.html">
+            <span class="std std-ref">Langmuir probe data analysis</span>
+          </a>
+        </span>
+      </p>
     </div>
+  </div>
+  <div class="sphx-glr-clear"></div>
```

Comment:
Why not just use the `.. nbsphinx::` directive for this?

Reply:
I tried and it started populating the TOC with these three examples.

Reply:
If you happen to know a way to do a nbsphinx directive without the TOC entries, it'd be much appreciated!

Reply:
Try `:hidden`

Reply:
Turns out :hidden: shows its contents in the TOC and hides the display on the main page, see the screenshot:

![image](https://user-images.githubusercontent.com/11289391/82408792-a72b4e00-9a6c-11ea-9d4f-7f5a0e759db4.png)

There's an extra duplicated TOC in there that's not seen in the main page.

Reply:
This could be an option...https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#confval-includehidden.  This would prevent hidden `toctree`'s from adding to the sidebar.  The trade-off, we can no longer hide our `toctree`'s in the main `index` page.

I think both ways have comparable PROs and CONs that make them equally nice and annoying.  I'll leave the choice up to you.

Reply:
Let's tinker with it another day, if that's all right with you. :) I'd love to apply a fix that doesn't propagate elsewhere to this, but I won't lose sleep over skipping it for now.